_grab_ui.py

```diff
@@ -460,6 +460,9 @@ def initialize(self) -> None:
         self.cmd.set_window_title('Grab – {}'.format(self.args.title))
         self._redraw()
 
+    def on_eot(self) -> None:
+        self.on_key_event(KeyEvent(mods=kk.CTRL, ctrl=True, key='d'))
+
```

Comment:
Instead, please set the new `perform_default_key_actions` variable introduced [here](https://github.com/kovidgoyal/kitty/commit/31fe35cd38bcc8009e95b3ec6e998a61f5237469) to `False`, and document that Ctrl+D is going to work only in Kitty ≥0.26.2.